packages/core/src/JWPlugin.ts

```diff
@@ -1,17 +1,26 @@
 import JWEditor from './JWEditor';
 import { ParsingFunction } from './Parser';
-import { CommandIdentifier, CommandDefinition, CommandHandler } from './Dispatcher';
+import { CommandIdentifier, CommandDefinition, CommandHandler, CommandArgs } from './Dispatcher';
 
 export interface JWPluginConfig {
     name?: string;
 }
 
+export interface Shortcut {
+    platform?: string;
+    pattern: string;
+    commandId: CommandIdentifier;
+    commandArgs?: CommandArgs;
+    priority?: number;
+}
+
 export class JWPlugin {
     static readonly parsingFunctions: Array<ParsingFunction> = [];
     name: string;
     editor: JWEditor;
     commands: Record<CommandIdentifier, CommandDefinition> = {};
     commandHooks: Record<CommandIdentifier, CommandHandler> = {};
+    defaultShortcuts: Shortcut[];
```

Comment:
I would have called it `shortcuts`. It is default by construction in the sense that they might be overriden later, but it's still the plugin's shortcut. You can never change the plugin shortcuts, what you can change is the *editor* shortcuts.